utils/windows_configuration.py

```diff
@@ -32,17 +32,49 @@ def get_registry_conf(agentConfig):
 
     return registry_conf
 
+def get_sdk_integration_path(hkey, reg_path):
```

Comment:
Mainly, for flexibility... you might call this as hkey being the key to `_winreg.HKEY_LOCAL_MACHINE` and have `reg_path` be `foo.bar`, or you could have hkey be the key to `_winreg.HKEY_LOCAL_MACHINE.foo` and reg_path=`bar`. I didn't see a reason to be more specific then I needed to.